palettes/palettes.json

```diff
@@ -8097,6 +8097,42 @@
     "#ffcb8b",
     "#ffffff"
   ],
+  "Oxocarbon Dark": [
+    "161616", 
```

Comment:
If you could add the `#` to the hex codes, so that it can compile 🙏

Reply:
Apologies, should be good now.